src/components/ApplicationGroup/Details/TriggerView/EnvTriggerView.tsx

```diff
@@ -2088,18 +2111,74 @@ export default function EnvTriggerView({ filteredAppIds, isVirtualEnv }: AppGrou
         )
     }
 
+    const renderCDModalHeader = () => (
+        <div className="trigger-modal__header flex right">
+            <button type="button" className="dc__transparent" onClick={closeCDModal}>
```

Comment:
Not yet replacing with <Button /> since close is also getting attached from various other places like Bulk, trigger view, env trigger view, etc